### ISSUE TITLE
go get error on Mac 

### ISSUE THREAD

**Issue by ookangzheng** [cloudflare/cf-terraforming#228]
I try to install on my Mac(11.2.3) with Go 1.16, and I got some error below

```
GO111MODULE=on go get -u github.com/cloudflare/cf-terraforming/...
# github.com/cloudflare/cf-terraforming/internal/app/cf-terraforming/cmd
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/access_application.go:50:65: not enough arguments in call to api.ZoneLevelAccessApplications
	have (string, cloudflare.PaginationOptions)
	want (context.Context, string, cloudflare.PaginationOptions)
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/access_policy.go:83:73: not enough arguments in call to api.ZoneLevelAccessApplications
	have (string, cloudflare.PaginationOptions)
	want (context.Context, string, cloudflare.PaginationOptions)
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/access_policy.go:95:58: not enough arguments in call to api.ZoneLevelAccessPolicies
	have (string, string, cloudflare.PaginationOptions)
	want (context.Context, string, string, cloudflare.PaginationOptions)
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/access_rule.go:56:48: not enough arguments in call to api.ListZoneAccessRules
	have (string, cloudflare.AccessRule, int)
	want (context.Context, string, cloudflare.AccessRule, int)
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/account_member.go:35:47: not enough arguments in call to api.AccountMembers
	have (string, cloudflare.PaginationOptions)
	want (context.Context, string, cloudflare.PaginationOptions)
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/record.go:206:53: cannot use record.Priority (type *uint16) as type int in argument to strconv.Itoa
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/record.go:208:59: cannot use record.Proxied (type *bool) as type bool in argument to strconv.FormatBool
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/root.go:194:31: not enough arguments in call to api.ZoneDetails
	have (string)
	want (context.Context, string)
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/root.go:202:29: cannot use zoneName (type string) as type context.Context in argument to api.ListZones:
	string does not implement context.Context (missing Deadline method)
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/root.go:216:29: not enough arguments in call to api.ListZones
	have ()
	want (context.Context, ...string)
go/pkg/mod/github.com/cloudflare/cf-terraforming@v0.0.0-20210315164413-e9a532ba267b/internal/app/cf-terraforming/cmd/account_member.go:35:47: too many errors
```

**Comment by jacobbednarz** [comment]
It looks like the README has steered you wrong here as `go get -u` will fetch and update the dependencies of the project.

> The -u flag instructs get to update modules providing dependencies
> of packages named on the command line to use newer minor or patch
> releases when available.

In this case, you don't want the `-u` as the breaking changes from 0.14 have not yet been ported into cf-terraforming.

**Comment by jacobbednarz** [comment]
this will be addressed with some upcoming changes in #231
